src/index.html

```diff
@@ -11,7 +11,180 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+      autocomplete="off"
+      onsubmit="onSubmit()"
+      class="main-form"
+    >
+
+      <fieldset class="main-form">
+        <legend>Personal Information</legend>
+
+        <div class="form-field">
+          <label for="surname">Surname:</label>
+          <input
+            type="text"
+            id="surname"
+            name="surname"
+          >
+        </div>
+
+        <div class="form-field">
+          <label for="name">Name:</label>
+          <input
+            type="text"
+            id="name"
+            name="name"
+          >
+        </div>
+
+        <div class="form-field">
+          <label for="how-old">How old are You?</label>
+          <input
+            type="number"
+            name="old"
+            id="how-old"
+            value="12"
+            min="0" max="100"
```

Comment:
Each attribut should be on a new line, also how can 0 year old use your website?
```suggestion
            min="12"
            max="100"
```